src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Color getRandomColor() {
+        int colorNumber = new Random().nextInt(Color.values().length);
```

Comment:
Do we really need to create a new instance of `Random`  every time when this method is called?
It is better to create a new Random field as class property.

Reply:
Fixed